test/e2e/monitoring_stack_controller_test.go

```diff
@@ -116,6 +115,9 @@ func TestMonitoringStackController(t *testing.T) {
 	}, {
 		name:     "Verify ability to scale down Prometheus",
 		scenario: prometheusScaleDown,
+	}, {
+		name:     "Verify ability to scale down Prometheus",
+		scenario: prometheusScaleDown,
```

Comment:
unexpected addition?

Reply:
Seems like it, @marioferh let's remove it :D

Reply:
done